src/main/java/frc/robot/RobotContainer.java

```diff
@@ -132,6 +151,10 @@ public void onExitDisabled() {
     signalingSubsystem.clearLEDs();
   }
 
+  private Command autonTest() {
+    return new InstantCommand(() -> SmartDashboard.putBoolean("NamedCommand test", true));
+  }
+
```

Comment:
getAutonomousCommand() below needs to get updated to run the command selected by the autoChooser